### ISSUE TITLE
"DiffEqArray Indexing Tests" from RecursiveArrayTools not included in downstream testing

### ISSUE THREAD

**Issue by ArnoStrouwen** [SciML/SymbolicIndexingInterface.jl#31]
Not sure if it is intentional that these tests are not included:
https://github.com/SciML/RecursiveArrayTools.jl/blob/0e877e32b89dd13159c11b1ff4eeba4aa3b80302/test/runtests.jl#L63-L64

**Comment by ChrisRackauckas** [comment]
@AayushSabharwal was this addressed?

**Comment by AayushSabharwal** [comment]
No, I'll PR to fix it. It looks like we run Core RAT in downstream tests which isn't particularly useful.
